presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationMetadata.java

```diff
@@ -222,8 +231,13 @@ public static int countInputChannels(List<ParameterMetadata> metadatas)
     {
         int parameters = 0;
         for (ParameterMetadata metadata : metadatas) {
-            if (metadata.getParameterType() == INPUT_CHANNEL || metadata.getParameterType() == NULLABLE_INPUT_CHANNEL) {
-                parameters++;
+            switch (metadata.getParameterType()) {
```

Comment:
I think the if was more clear, lets change it back. The switch with fall throughs and only one case seems weird